src/main/java/org/sonarsource/sonarlint/ls/AnalysisManager.java

```diff
@@ -212,22 +228,23 @@ private void analyze(URI fileUri, boolean shouldFetchServerIssues) {
 
     Optional<ProjectBindingWrapper> binding = bindingManager.getBinding(fileUri);
     AnalysisResultsWrapper analysisResults;
+    final Optional<GetJavaConfigResponse> javaConfigOptFinal = javaConfigOpt;
```

Comment:
Why not make it final from the beginning?

Reply:
Good catch, that's a pre-refactoring remain